com.photopea.Photopea.yml

```diff
@@ -0,0 +1,22 @@
+app-id: com.photopea.Photopea
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: photopea
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --share=network
+  - --filesystem=xdg-pictures
+modules:
+  - name: photopea
+    buildsystem: simple
+    build-commands:
+    - install -Dm755 -t /app/bin photopea
```

Comment:
you copy a binary (which I assume is x86_64 only) but build a flatpak on aarch64 too (it's by default).

Wouldn't it be better in that case to build from source?